core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala

```diff
@@ -62,6 +62,10 @@ trait ExternalBackendConf extends SharedBackendConf {
   def kerberosPrincipal = sparkConf.getOption(PROP_EXTERNAL_KERBEROS_PRINCIPAL._1)
   def kerberosKeytab = sparkConf.getOption(PROP_EXTERNAL_KERBEROS_KEYTAB._1)
   def runAsUser = sparkConf.getOption(PROP_EXTERNAL_RUN_AS_USER._1)
+  def externalH2ODriverIf = sparkConf.getOption(PROP_EXTERNAL_DRIVER_IF._1)
```

Comment:
WDYT about `sparkConf.getOption(PROP_EXTERNAL_DRIVER_IF._1, PROP_EXTERNAL_DRIVER_IF._2)`? I know that in both cases `None` is returned, but if 
```val PROP_EXTERNAL_DRIVER_IF = ("spark.ext.h2o.external.driver.if", None)``` specifies a default value...

Reply:
If I understand the motivation, this is to ensure we are prepared for cases when we change the default value of the underlying option & forget to change the getter?

The idea makes sense, however the code `getOption` does not accept second argument and `get` methods does not accept Option as value. Given these ^, I would keep it as it is. WDYT?

Reply:
Ok, let's leave it as it is.